cpp/ql/test/library-tests/dataflow/taint-tests/default_taint.cpp

```diff
@@ -0,0 +1,62 @@
+void alias_taint(const char *a, const char** pa, const char* alias_a)
```

Comment:
The `localTaint.ql` is failing because you added `default_taint.cpp` in its directory, and it picks up results from all files in that directory. I suggest putting this new test in a new directory `cpp/ql/test/library-tests/dataflow/DefaultTaintTracking`, which is where I'm putting the test for #2657.

Reply:
I've moved the test so that it's now in the `DefaultTaintTracking` folder.